File: fos-client/src/app/models/food.ts


    // $Classes/Enums/Interfaces(filter)[template][separator]
    // filter (optional): Matches the name or full name of the current item. * = match any, wrap in [] to match attributes or prefix with : to match interfaces or base classes.
    // template: The template to repeat for each matched item
    // separator (optional): A separator template that is placed between all templates e.g. $Properties[public $name: $Type][, ]

    // More info: http://frhagn.github.io/Typewriter/

    
          import { Photo } from './photo';
import { Price } from './price';
import { DiscountPrice } from './discount-price';

    export class Food    {
           
        public description: string = null;   
        public id: number = 0;   
        public is_available: boolean = false;   
        public is_group_discount_item: boolean = false;   
        public photos: Photo[] = [];   
        public price: Price = null;   
        public discount_price: DiscountPrice = null;   
        public name: string = null;   
        public time: string = null;   
        public options: string = null;   
        public properties: string = null;   
        public quantity: string = null;
    }
         

File: fos-api/FOS/FOS.Model/Domain/NowModel/Food.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FOS.Model.Domain.NowModel
{
    public class Food
    {
        [JsonProperty("description")]
        public string Description { get; set; }
        [JsonProperty("id")]
        public int Id { get; set; }
        [JsonProperty("is_available")]
        public bool IsAvailable { get; set; }
        [JsonProperty("is_group_discount_item")]
        public bool IsGroupDiscountItem { get; set; }
        [JsonProperty("photos")]
        public List<Photo> Photos { get; set; }
        [JsonProperty("price")]
        public Price Price { get; set; }
        [JsonProperty("discount_price")]
        public DiscountPrice DiscountPrice { get; set; }
        [JsonProperty("name")]
        public string Name { get; set; }
        [JsonProperty("time")]
        public string Time { get; set; }
        [JsonProperty("options")]
        public string Options { get; set; }
        [JsonProperty("properties")]
        public string Properties { get; set; }
        [JsonProperty("quantity")]
        public string Quantity { get; set; }

    }
}
